server/controllers/bookmark.controller.js

```diff
@@ -62,5 +62,43 @@ const toggleBookmark = async (req, res) => {
   }
 };
 
-const bookmarkController = { toggleBookmark };
+const userGetTheirBookmarkArticles = async (req, res) => {
```

Comment:
You can name this `getBookMarkedArticlesForUser`. `userGetTheirBookmarkArticles` is just awkward @AnayoOleru

Reply:
> You can name this `getBookMarkedArticlesForUser`. `userGetTheirBookmarkArticles` is just awkward @AnayoOleru

That's true. I've fixed it